### PR TITLE
turn off configurations for phase 2

### PR DESCRIPTION
**Pull request by heatherlogan-scottlogic**

### CODE REVIEW

File: frontend/src/components/DefenceBox/DefenceMechanism.tsx [hunk 1]
```diff
@@ -7,11 +7,13 @@ const { TiTick, TiTimes } = require("react-icons/ti");
 
 function DefenceMechanism({
   defenceDetail,
+  currentPhase,
   setDefenceActive,
   setDefenceInactive,
   setDefenceConfiguration,
 }: {
   defenceDetail: DefenceInfo;
+  currentPhase: number;
```
Comment (gsproston-scottlogic) [inline frontend/src/components/DefenceBox/DefenceMechanism.tsx:16]: I'd personally prefer this to be a boolean called something like `showConfiguration`. That way the component doesn't need to know about phases.

**Review by gsproston-scottlogic — CHANGES_REQUESTED** [review]
This works for me, though I think it'd be better if we passed down a boolean for showing defences rather than the phase number.

File: frontend/src/components/DefenceBox/DefenceMechanism.tsx (hunk 1 above)
Comment (heatherlogan-scottlogic) [reply frontend/src/components/DefenceBox/DefenceMechanism.tsx:16]: sure can do that

**Review by gsproston-scottlogic — APPROVED** [review]
Perfect thank you!

**Pull request merged**